src/bench/block_assemble.cpp

```diff
@@ -99,6 +99,7 @@ static void AssembleBlock(benchmark::State& state)
     }
     for (const auto& txr : txs) {
         CValidationState state;
+        LOCK(::cs_main);
```

Comment:
Could just take the lock in the first line of the benchmark once?

Reply:
Fixed.